core-ui/src/shared/components/MonacoEditorESM/Editor.scss

```
@@ -1,9 +1,8 @@
 .resource-form__wrapper {
   display: flex;
   flex-direction: column;
-  height: calc(100% - 21px - 36px);
+  flex-grow: 2;
```

why 2? i don't think it is in relation with other `flex-glow` items? or is it? `1` would be sufficient here.
```suggestion
flex-grow: 1;
```